manager/service.go

```diff
@@ -156,3 +170,12 @@ func (ms *managerService) publishEvent(event, cmpID, status string, details json
 		},
 	}
 }
+
+func getComputationHash(ac *agent.Computation) ([32]byte, error) {
```

Comment:
Rename to `computationHash` and use like `ch := computationHash(ac)`. If there is no need to pass the pointer param here (i.e. we do not change the computation from this function), pass by value instead.

Reply:
Done. The computation is passed by value now.